tests/2-doubleClickButton.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { test } from "playwright/test";
+
+test('Double click on button and verify message', async ({ page }) => {
+
+  await page.goto('/buttons');
```

Comment:
I'd recommend to add navigations method instead of going directly to specific path